element_data_loader/path_utils.py

```diff
@@ -0,0 +1,57 @@
+import datajoint as dj
+import pathlib
+
+
+dj.config['enable_python_native_blobs'] = True
```

Comment:
This is set by default. You do not need to set it but you could assert it if you like.

Reply:
Thanks.  Removed.